### PR TITLE
Add latest jdk

### PR DESCRIPTION
**Pull request by jbescos**
Signed-off-by: Jorge Bescos Gascon <jorge.bescos.gascon@oracle.com>

### CODE REVIEW

File: wsit/ws-sx/wssx-impl/pom.xml [hunk 1]
```diff
@@ -70,6 +70,17 @@
                 </executions>
 -->
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.13</version>
```
Comment (lukasj) [inline wsit/ws-sx/wssx-impl/pom.xml]: this version is too old and there should be one defined already in parent pom

File: wsit/ws-tx/pom.xml [hunk 2]
```diff
@@ -35,4 +35,13 @@
         <module>wstx-services</module>
     </modules>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>3.3.1</version>
```
Comment (lukasj) [inline wsit/ws-tx/pom.xml]: plugin version belongs to parent's pom plugin management section

**Comment by jbescos** [comment]
I have done the requested changes.

To make it work in GitHub actions I had to remove the 'verify'. It provides little information of the error, and I am not able to reproduce it locally with the command:

`mvn -B -V -U -C -Pstaging,oss-release clean verify org.glassfish.copyright:glassfish-copyright-maven-plugin:check -Dgpg.skip=true`

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:jar (attach-javadocs) on project wsrm-impl: MavenReportException: Error while generating Javadoc: 
Error:  Exit code: 1 - error: An internal exception has occurred. 
Error:    	(java.lang.NullPointerException: Cannot invoke "com.sun.source.util.DocTreePath.getTreePath()" because "path" is null)
Error:  Please file a bug against the javadoc tool via the Java bug reporting page
Error:  (http://bugreport.java.com) after checking the Bug Database (http://bugs.java.com)
Error:  for duplicates. Include error messages and the following diagnostic in your report. Thank you.
Error:  java.lang.NullPointerException: Cannot invoke "com.sun.source.util.DocTreePath.getTreePath()" because "path" is null
Error:  	at jdk.javadoc/jdk.javadoc.internal.tool.JavadocLog.getDiagnosticSource(JavadocLog.java:618)
Error:  	at jdk.javadoc/jdk.javadoc.internal.tool.JavadocLog.print(JavadocLog.java:245)
Error:  	at jdk.javadoc/jdk.javadoc.internal.doclets.toolkit.Messages.report(Messages.java:187)
Error:  	at jdk.javadoc/jdk.javadoc.internal.doclets.toolkit.Messages.warning(Messages.java:133)
Error:  	at jdk.javadoc/jdk.javadoc.internal.doclets.formats.html.HtmlDocletWriter.seeTagToContent(HtmlDocletWriter.java:1072)
Error:  	at jdk.javadoc/jdk.javadoc.internal.doclets.formats.html.HtmlDocletWriter$2.visitLink(HtmlDocletWriter.java:1557)
Error:  	at jdk.javadoc/jdk.javadoc.internal.doclets.formats.html.HtmlDocletWriter$2.visitLink(HtmlDocletWriter.java:1419)
```

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**